pages/data-validation.js

```diff
@@ -46,7 +46,8 @@ export class DataValidation extends Component {
   }
 
   checkEligibiltyPaths(b) {
-    return !(b.eligibilityPaths && b.eligibilityPaths != "");
+    console.log(b.elibigbilityPaths);
+    return !(b.elibigbilityPaths && b.elibigbilityPaths != "");
```

Comment:
Wait, I thought I fixed that in AirTable? We should fix the AirTable typo and use the correct spelling here.